projects/novo-elements/src/elements/date-time-picker/DateTimePicker.ts

```diff
@@ -224,15 +231,15 @@ export class NovoDateTimePickerElement implements ControlValueAccessor {
   writeValue(model: any): void {
     this.model = model;
     if (Helpers.isEmpty(model)) {
-      this.model = new Date();
+      this.model = this.createFullDateValue(this.datePickerValue, this.timePickerValue);
```

Comment:
wouldn't this be a change in behavior for existing date time pickers?

Reply:
Changed it back. I had updated it to fix something but it did not change the behavior.